.travis/dot.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Travis already has a function which does this -- called something like `travis_wait`...

Reply:
Also `travis_jigger` it seems -- https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/bash/travis_wait.bash

Reply:
https://github.com/chanzuckerberg/go-travis-wait and https://pypi.org/project/travis-wait-improved/ might be other options too...

Reply:
Thanks for suggestions. I avoided `travis_wait` because it does not pipe the output during building. I'll look into `jigger` and `improved` version of `travis_wait`

Reply:
@kowalewskijan -- This should probably be something in the common CI scripts too.

Reply:
I think we can override default `travis_wait` and `travis_jigger` like here: https://github.com/matomo-org/travis-scripts/blob/master/travis-helper.sh and we can keep these overridden functions in `common.sh`.

Reply:
I added overridden functions